### PR TITLE
Fixing nav margins

### PR DESCRIPTION
**Pull request by edolyne**
@jbaxleyiii and I were looking over positioning of icons.  It doesn't look like the native app build will need the negative margin.  We will need to view this change on multiple devices for the web version, but this appears to fix a number of ui oddities across the app.

### CODE REVIEW

File: stylesheets/_hacks.scss [hunk 1]
```diff
@@ -106,3 +106,8 @@ blockquote p {
 .plain {
   text-decoration: none;
 }
+
+
+.floating:before {
+  margin-right: 0!important; // XXX JAMES SAID THIS IS OKAY
```
Comment (johnthepink) [inline stylesheets/_hacks.scss]: Is this okay to add to all floating elements? I'm sure we are using that that class in more places than the nav. Maybe we should apply this only to `.floating` inside the nav?

**Comment by johnthepink** [comment]
@edolyne sweet! I added a note, and I'll check this out in a bit.

File: stylesheets/_hacks.scss (hunk 1 above)
Comment (jbaxleyiii) [inline stylesheets/_hacks.scss]: @johnthepink that was something we used to have to do in junction but I'm not sure if it is still needed anymore. I'm pretty sure it was added to combat whitespace issues which aren't an issue anymore. I'd like to look around the app but I feel pretty good we can add this, and remove the hack from junction

eventually flexbox 👍

Comment (johnthepink) [inline stylesheets/_hacks.scss]: @jbaxleyiii sweet thanks for the info. I'll poke around as well

**Pull request closed without merging**